src/test/java/com/mavenit/argos/SmokeTestSuite.java

```diff
@@ -0,0 +1,286 @@
+        System.out.println("\n\n'verifySearchTitle' Test Passed - you can see search title name as: " + actualTitle + ", which matches to searched string as 'puma'\n\n");
+    }
+
+    @Test
+    public void verifyProductRating() throws InterruptedException{
+        // for example we have selected the products with 5 star rating
+        doSearch("Smart watches");
+        Thread.sleep(2000);
+        driver.findElement(By.cssSelector("div[data-facet=\"customer rating\"]")).click();
+        Thread.sleep(5000);
+        driver.findElement(By.cssSelector("label[data-e2e=\"5-link\"]")).click();
+        String selectedRating = "5.0";
+        Thread.sleep(5000);
+        System.out.println("'verifyProductRating' Test Passed - you can see all the below products has rating greater than or equal toe '5'");
+        List<WebElement> ratingWebElements = driver.findElements(By.cssSelector("div[data-test=\"component-ratings\"]"));
+        for (WebElement indProductRating : ratingWebElements) {
+            String currentRating = indProductRating.getAttribute("data-star-rating");
+            double currentRating1 = Double.parseDouble(currentRating);
+
+            // now to verify that previously selected rating is same for all current products
+            assertTrue("Current Products have different Rating ", Double.parseDouble(currentRating) > Double.parseDouble(selectedRating));
+            System.out.printf("\nThe Rating of individual product is :" + "%.2f", currentRating1);
+        }
+    }
+
+    @Test
+    public void basketTest() throws InterruptedException {
+
+        doSearch("tripods, monopods and cases");
+        Thread.sleep(4000);
+        List<WebElement> productWebElements = driver.findElements(By.cssSelector("a[data-test='component-product-card-title']"));
+        if (productWebElements.size() == 0) {
+            Assert.fail("No Products found with: " + "tripods, monopods and cases");
+        }
+
+        // TODO: 2020-02-08 this will be converted in future
+        Random random = new Random();
+        int randomNumber = random.nextInt(productWebElements.size()-1 );
+
+        WebElement selectedElement = productWebElements.get(randomNumber);
+        String selectedProductName = selectedElement.getText();
+        selectedElement.click();
+        Thread.sleep(2000);
+        addToBasket();
+        Thread.sleep(5000);
+        goToBasket();
+        Thread.sleep(2000);
+        String actual = getProductNameInBasket();
+        assertThat(actual, is(equalToIgnoringCase(selectedProductName)));
+
+        System.out.println("\n\n'basketTest' Test Passed - Actual selected product & the product in the basket is same");
+        System.out.println("\nTotal number of products are: " + productWebElements.size());
+        System.out.println("Index number of random product is : " + randomNumber);
+        System.out.println("\nActual selected product is: " + selectedProductName );
```

Comment:
Code should not contains System.out

Reply:
i was trying to parametrised this sout but unbale to do it. just using sout again and again to export the results on external result.txt file. i need your help in this in next session